pkg/users/authorized-keys-panel.js

```diff
@@ -150,10 +151,9 @@ export function AuthorizedKeys({ name, home, allow_mods }) {
                 <div className="pf-c-card__actions">
                     { allow_mods &&
                     <Button onClick={() => add_authorized_key_dialog(manager)}
-                        id="authorized-key-add">
-                        <span className="fa fa-plus" />
-                    </Button>
-                    }
+                            isSmall
+                            id="authorized-key-add"
+                            icon={<PlusIcon />} />}
```

Comment:
This does not seem to be aligned properly:
![iconout](https://user-images.githubusercontent.com/12330670/97989516-ef51e900-1dde-11eb-9d0d-7cbf3130287d.png)
The style that does that comes from PF though

Reply:
FIxed the alignment, but I can't figure out why it not square but it's squeezed. I will ask @garrett for help here

Reply:
There's a margin from the icon that isn't removed when it's the only content. This is causing the icon to not be centered.

```scss
.pf-c-button__icon.pf-m-start {
	margin-right: var(--pf-c-button__icon--m-start--MarginRight);
}
```

It should probably be something like:

```scss
.pf-c-button__icon.pf-m-start:not(:only-child) {
	margin-right: var(--pf-c-button__icon--m-start--MarginRight);
}
```

We could override it with a:

```scss
.pf-c-button__icon.pf-m-start:only-child {
	margin-right: 0;
}
```

---

And as far as rectangle versus square, we have this _(linebreaks and spacing added for clarity)_:

```scss
padding: 
   var(--pf-c-button--PaddingTop) 
   var(--pf-c-button--PaddingRight) 
   var(--pf-c-button--PaddingBottom) 
   var(--pf-c-button--PaddingLeft);
```

and defined by:

```scss
--pf-c-button--m-display-lg--PaddingTop: var(--pf-global--spacer--md);
--pf-c-button--m-display-lg--PaddingRight: var(--pf-global--spacer--xl);
--pf-c-button--m-display-lg--PaddingBottom: var(--pf-global--spacer--md);
--pf-c-button--m-display-lg--PaddingLeft: var(--pf-global--spacer--xl);
```

We'd want to change the padding of left and right to the same as the top to make it square (or very close to square, depending on the icon).

Reply:
Also: We need to have the string "Add public key" (or even just "Add key") on the button. It can either replace the + with the add string... or the string should be there in ARIA _and_ a tooltip.